doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -273,9 +277,9 @@ Strict access permissions must be set on OpenStack configuration files.
 
 **Configuration and management**
 
-Configuration operations must be traced. Events such as system access attempts, actions with high privileges, modification of configuration must be logged and exported on the fly to a distant storage. The communication channel used for log collection must be protected in integrity and confidentiality and logs protected against unauthorized modification.
+Configuration operations must be tracked (sec.mon.006, sec.mon.007). Events such as system access attempts, actions with high privileges, modification of configuration must be logged and exported on the fly to a distant storage. The communication channel used for log collection must be protected in integrity and confidentiality and logs protected against unauthorized modification (sec.mon.004).
 
-Per sec.LCM.002 requirement, management protocols limiting security risks must be used such as SNMPv3, SSH v2, ICMP, NTP, syslog and TLS. How to secure logging is described in the following section. 
+Per sec.lcm.002 requirement, management protocols limiting security risks must be used such as SNMPv3, SSH v2, ICMP, NTP, syslog and TLS. How to secure logging is described in the following section. 
 
 **Platform backup**
 
```

Comment:
We are missing this section on backup

Reply:
@iangardner22 the text states - should we add more details? if yes, please suggest.

**Platform backup**

The storage for backup must be independent of storage offered to tenants.

Reply:
Sorry my bad - I was looking at the sections that had been changed and of course these were not shown...   Looks fine